boards/risc-v/esp32c3/esp32c3-devkit/src/esp32c3_spiflash.c

```diff
@@ -42,6 +42,14 @@
 #include "esp32c3_spiflash.h"
 #include "esp32c3-devkit.h"
 
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define ESP32C3_MTD_PATH      "/dev/esp32c3flash"
+
+#define ESP32C3_FS_MOUNT_PTR  CONFIG_ESP32C3_SPIFLASH_FS_MOUNT_PT
```

Comment:
```suggestion
#define ESP32C3_FS_MOUNT_PT   CONFIG_ESP32C3_SPIFLASH_FS_MOUNT_PT
```
nit: This one should also be fixed